docs/client-linux-wireguard.md

```diff
@@ -2,14 +2,16 @@
 
 ## Install WireGuard
 
-To connect to your AlgoVPN using [WireGuard](https://www.wireguard.com) from Ubuntu, first install WireGuard:
+To connect to your AlgoVPN using [WireGuard](https://www.wireguard.com) from Ubuntu, make sure your system is up-to-date then install WireGuard:
 
 ```shell
-# Ubuntu 16.04 only: Add the WireGuard repository
-sudo add-apt-repository ppa:wireguard/wireguard
-sudo apt update
+# Update your system:
+sudo apt update && sudo apt upgrade
 
-# Install the tools:
+# If the file /var/run/reboot-required exists then reboot:
+sudo reboot
```

Comment:
`[ -e /var/run/reboot-required ] && sudo reboot`

Reply:
I had a feeling you were gonna say that... :grinning: